tools/binexport.ps1

```diff
@@ -0,0 +1,52 @@
+<#
```

Comment:
Recommend updating the top level README to use `binexport.ps1` if you prefer folks to use it over the `binexport.cmd`

Reply:
Done.